### PR TITLE
Add camera choice option to QR scanner field

### PR DESCRIPTION
**Pull request by zedascouves**
## Description
_Allows selecting the camera for the QR scanner field. Issue https://github.com/Budibase/budibase/issues/11459._

Addresses: 
- `https://github.com/Budibase/budibase/issues/11459`

## Documentation
- [X] I have reviewed the budibase documentatation to verify if this feature requires any changes. If changes or new docs are required I have written them.

### CODE REVIEW

**Review by aptkingston — APPROVED** [review]
Looks good to me! Thanks for the contribution @zedascouves.

File: packages/client/src/components/app/forms/CodeScanner.svelte [hunk 1]
```diff
@@ -20,7 +21,7 @@
   let cameraEnabled
   let cameraStarted = false
   let html5QrCode
-  let cameraSetting = { facingMode: "environment" }
+  let cameraSetting = { facingMode: {preferredCamera} }
```
Comment (aptkingston) [inline packages/client/src/components/app/forms/CodeScanner.svelte:24]: This is now an object and won't work - we just want a string.

This needs to be:
```js
let cameraSetting = { facingMode: preferredCamera }
```

**Review by aptkingston — CHANGES_REQUESTED** [review]

**Review by zedascouves — COMMENTED** [review]
I've changed the code as suggested.

**Review by aptkingston — APPROVED** [review]

**Comment by zedascouves** [comment]
Do I need to do anything else for this to be successfully merged?

**Pull request merged**